### DISCUSSION TITLE
Can't get to invite

### DISCUSSION THREAD

**Discussion by dsm1212** [TandoorRecipes/recipes#2007] [General]
Ok, I'm at a loss to find my way back to invite. The docs mention doing invites from the admin account, but later say you cannot invite from admin. But my user account doesn't have "System" where it says the invite option is anymore. I did invites months ago so it had at one time. Do I need to create a superuser account now?

**Comment by smilerz** [comment]
you can send an invite from the Space Settings screen

**Reply by dsm1212** [reply]
Doh! There it is. Thanks
